data-prepper-plugins/service-map-stateful/build.gradle

```diff
@@ -16,10 +16,12 @@ dependencies {
     compile project(':data-prepper-plugins:common')
     compile project(':data-prepper-plugins:lmdb-prepper-state')
     compile project(':data-prepper-plugins:mapdb-prepper-state')
+    testCompile project(':data-prepper-api').sourceSets.test.output
+    implementation "io.micrometer:micrometer-core:1.5.1"
     implementation "com.fasterxml.jackson.core:jackson-databind:${versionMap.jackson_databind}"
     implementation "io.opentelemetry:opentelemetry-proto:${versionMap.opentelemetry_proto}"
     testImplementation "org.hamcrest:hamcrest:${versionMap.hamcrest}"
-    testImplementation "org.mockito:mockito-core:${versionMap.mockito}"
+    testImplementation "org.mockito:mockito-inline:3.6.28"
```

Comment:
Might be time to move mockito-inline to the version map (I see there are 3 other uses of it). If it's not trivial can be a separate effort.